File: DownloadFiles/TGNet/Chapter10/C#/Exercise10-2/WebServTestBed/WebServTestBed/Form1.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Text;
using System.Windows.Forms;

namespace WebServTestBed
{
    public partial class Form1 : Form
    {
        public Form1()
        {
            InitializeComponent();
        }


        private void button1_Click(object sender, EventArgs e)
        {
            CalculateArea.Service AreaServ = new CalculateArea.Service();
            float sngLength;
            float sngWidth;
            float sngArea;
            sngLength = Convert.ToSingle(txtLength.Text);
            sngWidth = Convert.ToSingle(txtWidth.Text);
            sngArea = AreaServ.RectangleArea(sngLength, sngWidth);
            lblArea.Text = sngArea.ToString();

        }
    }
}

File: DownloadFiles/TGNet/Chapter10/VB/Exercise10-2/WebServTestBed/WebServTestBed/Form1.vb
Public Class Form1

    Private Sub Label1_Click(ByVal sender As System.Object, ByVal e As System.EventArgs) Handles lblArea.Click

    End Sub

    Private Sub Button1_Click(ByVal sender As System.Object, ByVal e As System.EventArgs) Handles Button1.Click

        Dim AreaServ As New CalculateArea.Service
        Dim sngLength As Single
        Dim sngWidth As Single
        Dim sngArea As Single

        sngLength = Convert.ToSingle(txtLength.Text)
        sngWidth = Convert.ToSingle(txtWidth.Text)
        sngArea = AreaServ.RectangleArea(sngLength, sngWidth)

        lblArea.Text = sngArea.ToString

    End Sub
End Class
